content/swiftui/concepts/viewmodifier/terms/tint/tint.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '.tint()'
+Description: 'Sets the tint color displayed in a view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.tint()`** modifier sets the tint color displayed in a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+Button("This is a button")
+    .tint(color)
+```
+
+The `.tint()` applies the `color` to the `Button` which conforms to the `View` protocol.
+
+## Example
+
+```swift
+struct ContentView: View {
+    var body: some View {
+        Button("This is a button")
+            .tint(.blue)
+    }
+}
+```
+
+In the above example, the `.tint()` modifier with the `.blue` parameter is called on the `Button` view. This applies the blue tint color to the button.
+
+This will display the following:
+
+![SwiftUI ViewModifier .tint()](https://raw.githubusercontent.com/Codecademy/docs/main/media/SwiftUI-tint-display.png)
```

Comment:
use better desc for this image. If somehow this image fails to load then this will appear instead.